core/src/test/java/edu/wpi/grip/util/GRIPCoreTestModule.java

```diff
@@ -69,6 +74,12 @@ public void tearDown() {
     protected void configure() {
         assert setUp : "The GRIPCoreTestModule handler was not set up. Call 'setUp' before passing the injector";
         super.configure();
+        // HTTP server injection bindings
+        bind(GripServer.HttpServerFactory.class).to(GripServerTest.TestServerFactory.class);
+        bind(GripServer.class).asEagerSingleton();
+        bind(MapNetworkPublisherFactory.class)
+                .annotatedWith(Names.named("httpManager"))
+                .to(HttpManager.class);
```

Comment:
https://github.com/WPIRoboticsProjects/GRIP/pull/551/files#r57812557

Also, do you need the entire server?
It seems like the HTTPSource only needs a class that provides the `addPostHandler` and `removePostHandler` methods. You could bind that interface to the instance for the GRIPNetworkModule and in the test module just use a mock.